.changeset/nasty-seahorses-confess.md

```diff
@@ -0,0 +1,15 @@
+---
+'@vocab/phrase': minor
+'@vocab/core': minor
+---
+
+Vocab pull: support custom key
+
+```json
+{
+  "thanks": {
+    "message": "Thanks",
+    "global-key": "app.thanks.label"
+  }
+}
+```
```

Comment:
Generally we try and put a bit more detail/description into the changeset, as well as an example if appropriate so that consumers don't _have_ to read the README to understand what the new API does and how to use it.